src/cpu/x64/jit_avx512_core_amx_conv_utils.hpp

```diff
@@ -283,6 +283,7 @@ inline void execute_backward_convolution_body(const exec_ctx_t &ctx,
                 p.dst_scale = &dst_scales[0];
                 p.acc_s32 = wsp + ithr * jcp.wsp_buffer_size;
                 p.last_h = (ih + ih_step <= ih_e);
+                assert(iwb >= 0);
```

Comment:
Assert can't fix the issue for the user, if the issue is there.
What is the scenario when `iwb` can become negative?